### PR TITLE
Configure PHE site www.screening.nhs.uk and its aliases

### PR DESCRIPTION
**Pull request by issyl0**
Request from: https://govuk.zendesk.com/tickets/952717.

### CODE REVIEW

**Comment by fatbusinessman** [comment]
That all looks sensible: thanks for taking the time to explain it to me.

**Comment by issyl0** [comment]
Thanks. Glad you understood my explanation!

File: data/transition-sites/phe_screen.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+site: phe_screen
+whitehall_slug: public-health-england
+homepage_title: 'UK National Screening Committee'
+homepage: https://www.gov.uk/government/organisations/public-health-england
+tna_timestamp: 20201010101010 # Stub timestamp - TBC post-TNA crawl
+host: www.screening.nhs.uk
```
Comment (jamiecobbett) [inline data/transition-sites/phe_screen.yml]: What about `screening.nhs.uk`?

Comment (issyl0) [inline data/transition-sites/phe_screen.yml]: #1083.

**Pull request merged**